app/models/statutory_instrument.rb

```diff
@@ -12,16 +13,26 @@ class StatutoryInstrument < Document
   ).freeze
 
   attr_accessor(*FORMAT_SPECIFIC_FIELDS)
+  attr_accessor :organisations, :primary_publishing_organisation
 
   def initialize(params = {})
     super(params, FORMAT_SPECIFIC_FIELDS)
+    @primary_publishing_organisation = params[:primary_publishing_organisation]
+    @organisations = params[:organisations]
   end
 
   def self.title
     "Statutory instrument"
   end
 
-  def primary_publishing_organisation
-    "fef4ac7c-024a-4943-9f19-e85a8369a1f3"
+  def links
+    super.merge(
+      organisations: organisations,
```

Comment:
Since `organisations` should also contain the primary publishing org we should probably union them here so we can provide the right list of orgs regardless of whether the user put the primary org in the additional field or not.